README.md

```diff
@@ -63,58 +63,54 @@ The CronJob calls the endpoint `/prisoner-index/queue-housekeeping` which is not
 
 ### Running
 
-`localstack` is used to emulate the AWS SQS and Elastic Search service.  
-Any commands in `localstack/setup-sns.sh` and `localstack/setup-es.sh` will be run when `localstack` starts, so this contains commands to create the appropriate queues.
-Localstack listens on two main ports - 4566 for sns and sqs and 4571 for elasticsearch.
+`localstack` is used to emulate the AWS SQS and Elastic Search service. Any commands in `localstack/setup-sns.sh` and `localstack/setup-es.sh` will be run when `localstack` starts, so this contains commands to create the appropriate queues. Localstack listens on two main ports - 4566 for sns and sqs and 4571 for elasticsearch.&nbsp;<sup><em>1</em></sup>
```

Comment:
`<sup><em>1</em></sup>` - interesting!  Could they be clickable links?

Reply:
yes, they could be.  See https://github.com/fefong/markdown_readme#some-links-for-more-in-depth-learning.  However I'm more of the opinion that the content would be better including here / in a windows section rather than down at the bottom of the file.